src/routing_table/mod.rs

```diff
@@ -227,15 +198,17 @@ pub enum OwnMergeState<T: Binary + Clone + Copy + Default + Hash + Xorable> {
 /// See the [crate documentation](index.html) for details.
 #[derive(Clone, Eq, PartialEq)]
 pub struct RoutingTable<T: Binary + Clone + Copy + Debug + Default + Hash + Xorable> {
-    // Minimum number of nodes we consider acceptable in a section
+    /// Minimum number of nodes we consider acceptable in a section
     min_section_size: usize,
-    // Name of node holding this table
+    /// Name of node holding this table
     our_name: T,
-    // Prefix of our section
+    /// Prefix of our section
     our_prefix: Prefix<T>,
-    // Members of our section, including our own name
+    /// Members of our section, including our own name
     our_section: BTreeSet<T>,
-    // Other sections (excludes our own) (TODO: rename)
+    /// Our prefix version.
+    our_version: u64,
```

Comment:
I tried it and it seemed to make the code more complicated overall.
Maybe I'll give it another go, but I'd rather leave that for a separate PR.